web/docs/understand/language/operators/tail.md

```diff
@@ -0,0 +1,27 @@
+# tail
+
+Limits the input to the last N results.
```

Comment:
This also needs a changelog entry in the feature category.

Reply:
I added a changelog entry.